supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -324,12 +324,22 @@ There is no functional difference between the first server that was installed an
 
 [[ingress]]
 ==== image:images/yes.png[yes] Ingress (noun)
-*Description*: In Red Hat OpenShift, _Ingress_  is an API object developers can use to expose services through an HTTP(S) aware load balancing and proxy layer through a public DNS entry. The Ingress resource might further specify TLS options and a certificate, or specify a public CNAME that the OpenShift Ingress Controller should also accept for HTTP and HTTPS traffic. An administrator typically configures their Ingress Controller to be visible outside the cluster firewall and might also add additional security, caching, or traffic controls on the service content.
+*Description*: In Red Hat OpenShift, _Ingress_  is an API object developers can use to expose services through an HTTP(S) aware load balancing and proxy layer through a public DNS entry. The `Ingress` resource might further specify TLS options and a certificate, or specify a public CNAME that the OpenShift Ingress Controller (`IngressController`) should also accept for HTTP and HTTPS traffic. An administrator typically configures their Ingress Controller to be visible outside the cluster firewall and might also add additional security, caching, or traffic controls on the service content.
 
 *Use it*: yes
 
 *Incorrect forms*:
+"Ingress" alone, when referencing the `Ingress` resource.
+*See also*:
+
+[[ingress]]
```

Comment:
```suggestion
[[ingress-adj]]
```
Each glossary entry needs to have a unique ID to ensure that anchor links work correctly.

Reply:
I was thinking actually maybe the meaning I'm trying to convey can actually be wrapped into one ingress entry, not creating a separate entry for the adjective use.